### DISCUSSION TITLE
Is it possible to use a custom token instead of $ for params?

### DISCUSSION THREAD

**Discussion by Renart-fox** [TanStack/router#1565] [Q&A] [answered]
Hello. I am using TFVC for source control and due to naming conventions, I cannot use a dollar sign in a path component.
Is it possible to circumvent this issue in the configuration of tsr ? I was not able to find anything about this in the documentation.

Using another source control tool is not possible.

**Comment by SeanCassiere** [comment, marked as answer]
Currently, we don't have the ability to customize the wildcard value being the `$` in the router-generator.

But this isn't the most impossible thing to implement either. You could file an issue for this as a feature request, I don't see much of reason why we wouldn't add this in.
